pages/index.css

```diff
@@ -6,6 +6,31 @@
 @import url(../blocks/page/page.css);
 @import url(../blocks/page/__container/page__container.css);
 
+/* Price Block */
+@import url(../blocks/price/price.css);
+@import url(../blocks/price/__toggles/price__toggles.css);
+@import url(../blocks/price/__toggles/-slider/price__toggles-slider.css);
+@import url(../blocks/price/__toggles/-slider/_current/price__toggles-slider_current.css.css);
```

Comment:
Good job on importing the CSS modifier after the element.

Reply:
@5hraddha Thank you😊